.github/workflows/end_to_end_test.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
     runs-on: [self-hosted, linux, x64, e2e-runner]
     steps:
       - name: Docker version
-        run: docker --version
+        run: docker version
```

Comment:
When I run this command locally I don't need sudo access. Should this be changed to a docker command, e.g. `docker images`, that would normally require sudo access?

Reply:
From my testing it seems `docker version` would fail without `sudo`:
```
yhaliaw@lima-dev:~$ docker version
Client:
 Version:           20.10.21
 API version:       1.41
 Go version:        go1.18.1
 Git commit:        20.10.21-0ubuntu1~22.04.2
 Built:             Thu Mar  2 18:49:07 2023
 OS/Arch:           linux/arm64
 Context:           default
 Experimental:      true
Got permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Get "http://%2Fvar%2Frun%2Fdocker.sock/v1.24/version": dial unix /var/run/docker.sock: connect: permission denied
yhaliaw@lima-dev:~$ echo $?
1
```
The exit code is 1, which should indict an error.

With `sudo` the exit code is 0:
```
yhaliaw@lima-dev:~$ sudo docker version
Client:
 Version:           20.10.21
 API version:       1.41
 Go version:        go1.18.1
 Git commit:        20.10.21-0ubuntu1~22.04.2
 Built:             Thu Mar  2 18:49:07 2023
 OS/Arch:           linux/arm64
 Context:           default
 Experimental:      true

Server:
 Engine:
  Version:          20.10.21
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.18.1
  Git commit:       20.10.21-0ubuntu1~22.04.2
  Built:            Wed Feb 15 15:26:57 2023
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          v1.6.19
  GitCommit:        1e1ea6e986c6c86565bc33d52e34b81b3e2bc71f
 runc:
  Version:          1.1.4
  GitCommit:        v1.1.4-0-g5fd4c4d1
 docker-init:
  Version:          0.19.0
  GitCommit:        
yhaliaw@lima-dev:~$ echo $?
0
```

Reply:
The `docker --version` and `docker version` are two different commands. The `docker --version` does not need sudo to work.

Reply:
You are right